code/src/styles/MessageElement.css

```diff
@@ -0,0 +1,64 @@
+@import url('https://fonts.googleapis.com/css2?family=Trispace&display=swap');
+
+* {
+  font-family: 'Trispace', sans-serif;
+  text-rendering: optimizeLegibility;
```

Comment:
Hmm what does this one do? Now I got curious!

Reply:
The browser emphasizes legibility over rendering speed and geometric precision. I'm not sure if I can tell... but I wanted to try it out. XD